google/cloud/monitoring_v3/query.py

```diff
@@ -460,12 +461,16 @@ def _build_query_params(self, headers_only=False, page_size=None):
             from this request. Non-positive values are ignored. Defaults
             to a sensible value set by the API.
         """
-        params = {"name": self._project_path, "filter_": self.filter}
+        params = {"name": self._project_path, "filter": self.filter}
 
+        end_timestamp = timestamp.Timestamp()
+        end_timestamp.FromDatetime(self._end_time)
         params["interval"] = types.TimeInterval()
-        params["interval"].end_time.FromDatetime(self._end_time)
+        params["interval"].end_time = end_timestamp
```

Comment:
I don't think constructing the timestamp proto is necessary. Proto plus should be able to marshal from a `datetime`, like so:
```python
params["interval"].end_time = self._end_time
```

Reply:
done